staker/mint_stake.gno

```diff
@@ -42,7 +57,7 @@ func MintAndStake(
 
 		// send it over to position to wrap
 		banker := std.GetBanker(std.BankerTypeRealmSend)
-		banker.SendCoins(consts.STAKER_ADDR, consts.POSITION_ADDR, std.Coins{{"ugnot", int64(ugnotSent)}})
+		banker.SendCoins(consts.STAKER_ADDR, consts.POSITION_ADDR, std.Coins{{Denom: "ugnot", Amount: int64(ugnotSent)}})
```

Comment:
I'm really not sure if its necessary

Reply:
Not required, but included for readability.